package/MDAnalysis/coordinates/base.py

```diff
@@ -1461,6 +1461,60 @@ def __repr__(self):
             natoms=self.n_atoms
         ))
 
+    def timeseries(self, asel=None, start=None, stop=None, step=None,
```

Comment:
This is a new feature for trajectory readers. It cannot go into a patch release. It has to be in 0.17.0.